akka-docs-dev/rst/java/code/docs/http/javadsl/server/directives/HostDirectivesExamplesTest.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+
+package docs.http.javadsl.server.directives;
+
+import java.util.Arrays;
+
+import akka.http.javadsl.model.StatusCodes;
+import akka.http.javadsl.server.*;
+import akka.http.javadsl.testkit.JUnitRouteTest;
+
+import org.junit.Test;
+
+public class HostDirectivesExamplesTest extends JUnitRouteTest {
+  @Test
+  public void testHost() {
+    //#host1
+    final Route matchListOfHosts = host(
+        Arrays.asList("api.company.com", "rest.company.com"),
+        completeWithStatus(StatusCodes.OK));
+
+    path("test").route(matchListOfHosts);
+    //#host1
+
+    //#host2
+    final Route shortOnly = host(hostname -> hostname.length() < 10,
+        completeWithStatus(StatusCodes.OK));
+
+    path("test").route(shortOnly);
```

Comment:
Here is corresponding scala documentation: http://doc.akka.io/docs/akka-stream-and-http-experimental/snapshot/scala/http/routing-dsl/directives/host-directives/host.html#host

I don't know if this path("test") makes sense?

The Scala documentation shows test code: `Get() ~> Host("rest.company.com") ~> route ~> check`